src/cheerio_crawler.js

```diff
@@ -0,0 +1,95 @@
+import rp from 'request-promise';
+import _ from 'underscore';
+import cheerio from 'cheerio';
+import { checkParamOrThrow } from 'apify-client/build/utils';
+import BasicCrawler from './basic_crawler';
+import { isPromise } from './utils';
+
+const DEFAULT_OPTIONS = {
+    requestFunction: ({ request }) => rp({ url: request.url, method: request.method, headers: request.headers }),
+    pageOpsTimeoutMillis: 300000,
+    requestOptions: {
+        ignoreSslErrors: true,
+    },
+};
+
+export default class CheerioCrawler {
+    constructor(opts) {
+        const {
+            requestFunction,
+            handlePageFunction,
+            pageOpsTimeoutMillis,
+            requestOptions,
+
+            // Autoscaled pool options
+            maxMemoryMbytes,
+            maxConcurrency,
+            minConcurrency,
+            minFreeMemoryRatio,
+            isFinishedFunction,
+
+            // Basic crawler options
+            requestList,
+            requestQueue,
+            maxRequestRetries,
+            maxRequestsPerCrawl,
+            handleFailedRequestFunction,
+        } = _.defaults(opts, DEFAULT_OPTIONS);
+
+        checkParamOrThrow(requestFunction, 'opts.requestFunction', 'Function');
+        checkParamOrThrow(handlePageFunction, 'opts.handlePageFunction', 'Function');
+        checkParamOrThrow(handleFailedRequestFunction, 'opts.handleFailedRequestFunction', 'Maybe Function');
+        checkParamOrThrow(requestOptions, 'opts.requestOptions', 'Object');
+
+        if (requestOptions.ignoreSslErrors) process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
Is this the only way to do this? Because this affects the process globally. I thought request library has parameter called strictSSL

Reply:
Passing `rejectUnauthorized = false` should be possible to achieve the same. I figured that since it's a global option in the crawler settings, it should apply globally as well.

But thinking about it again, that would make sense directly in the `crawler-actor` and not in the SDK so I'll change it back to `rejectUnauthorized`.

Reply:
Since this is in SDK, it shouldn't affect the process globally. Just use the `strictSSL` param and that's it